GOVERNANCE.md

```diff
@@ -67,20 +67,25 @@ If a maintainer is no longer interested or cannot perform the maintainer duties
 
 # Approving PRs
 
-PRs may be merged only after receiving at least two approvals (LGTMs) from Committers or Maintainers. However, maintainers can sidestep this rule under justifiable circumstances. For example:
+PRs may be merged only after receiving at least two approvals (LGTMs) from Committers or Maintainers. However, Maintainers can sidestep this rule under justifiable circumstances. For example:
 
 - If a CI tool is broken, may override the tool to still submit the change.
 - Minor typos or fixes for broken tests.
 - The change was approved through other means than the standard process.
 
 # Decision Making Process
 
-Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus model](https://communitymgt.fandom.com/wiki/Lazy_consensus). Community-level decisions must be brought to broader awareness via communications such as community meetings and slack channels.
+Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus model](https://communitymgt.fandom.com/wiki/Lazy_consensus). Community-level decisions such as:
+
+- RFC approvals (see below)
+- Modifying GOVERNANCE.md
+
+must be brought to broader awareness via communications such as community meetings and slack channels and supermajority (2/3) approval from Maintainters is required.
```

Comment:
```suggestion
Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus model(https://communitymgt.fandom.com/wiki/Lazy_consensus).
Community or project level decisions such as RFC submission, creating a new project, maintainer promotion,  and major updates on GOVERNANCE must be brought to broader awareness of the community via community meetings, GitHub discussions,  and slack channels. A supermajority (2/3) approval from Maintainers is required for such approvals.
```

Reply:
@yeya24 PTAL:)

Reply:
Thanks for the work. This looks perfect!